snippets/ajax-cart-template.liquid

```diff
@@ -2,47 +2,42 @@
+    <form action="/cart" method="post" novalidate class="cart">
+      <div class="ajaxcart__inner{% endraw %}{% unless additional_checkout_buttons %} ajaxcart__inner--has-fixed-footer{% endunless %}{% raw %}">
         {{#items}}
         <div class="ajaxcart__product">
           <div class="ajaxcart__row" data-id="{{id}}">
             <div class="grid">
-              <div class="grid__item large--two-thirds">
-                <div class="grid">
-                  <div class="grid__item one-quarter">
-                    <a href="{{url}}" class="ajaxcart__product-image"><img src="{{img}}" alt=""></a>
-                  </div>
-                  <div class="grid__item three-quarters">
-                    <a href="{{url}}" class="h4">{{name}}</a>
-                    {{#if variantion }}
-                    <p>{{variation}}</p>
-                    {{/if}}
-                  </div>
-                </div>
+              <div class="grid__item one-quarter">
+                <a href="{{url}}" class="ajaxcart__product-image"><img src="{{img}}" alt=""></a>
               </div>
-              <div class="grid__item large--one-third">
-                <div class="grid">
-                  <div class="grid__item one-third">
+              <div class="grid__item three-quarters">
+                <p>
+                  <a href="{{url}}" class="ajaxcart__product-name">{{name}}</a>
+                  {{#if variation}}
+                    <span class="ajaxcart__product-meta">{{variation}}</span>
+                  {{/if}}
+                  {% endraw %}{% if settings.cart_vendor_enable %}{% raw %}
+                    <span class="ajaxcart__product-meta">{{ vendor }}</span>
+                  {% endraw %}{% endif %}{% raw %}
+                </p>
+
+                <div class="grid--full display-table">
+                  <div class="grid__item display-table-cell one-half">
                     <div class="ajaxcart__qty">
                       <button type="button" class="ajaxcart__qty-adjust ajaxcart__qty--minus" data-id="{{id}}" data-qty="{{itemMinus}}">&minus;</button>
                       <input type="text" class="ajaxcart__qty-num" value="{{itemQty}}" min="0" data-id="{{id}}" aria-label="quantity" pattern="[0-9]*">
                       <button type="button" class="ajaxcart__qty-adjust ajaxcart__qty--plus" data-id="{{id}}" data-qty="{{itemAdd}}">+</button>
                     </div>
                   </div>
-                  <div class="grid__item one-third text-center">
-                    <p>{{price}}</p>
-                  </div>
-                  <div class="grid__item one-third text-right">
-                    <p>
-                      <small><a href="/cart/change?id={{id}}&amp;quantity=0" class="ajaxcart__remove" data-id="{{id}}">{% endraw %}{{ 'cart.general.remove' | t}}{% raw %}</a></small>
-                    </p>
+                  <div class="grid__item display-table-cell one-half text-right">
+                    {{price}}
```

Comment:
Can you use triple curlies to allow HTML?
Even some stores use HTML entities there (in their money formats, by default), which will be escaped.